karapace/karapacemetrics.py

```diff
@@ -0,0 +1,119 @@
+"""
```

Comment:
issue: On the naming of this module, we're already in the karapace namespace, let's simply name the module `metrics`? (Resulting name would be `karapace.metrics` instead of repeating `karapace.karapacemetrics`).

Reply:
fixed